diff --git a/node_modules/react-bootstrap/esm/Form.d.ts b/node_modules/react-bootstrap/esm/Form.d.ts
index e54eb28..a90380f 100644
--- a/node_modules/react-bootstrap/esm/Form.d.ts
+++ b/node_modules/react-bootstrap/esm/Form.d.ts
@@ -8,7 +8,7 @@ import FormText from './FormText';
 import Switch from './Switch';
 import { BsPrefixProps, BsPrefixRefForwardingComponent } from './helpers';
 declare const FormRow: BsPrefixRefForwardingComponent<"div", unknown>;
-export interface FormProps extends React.HTMLAttributes<HTMLElement>, BsPrefixProps {
+export interface FormProps extends React.HTMLAttributes<HTMLFormElement>, BsPrefixProps {
     inline?: boolean;
     validated?: boolean;
 }
